capture.py

```diff
@@ -126,9 +136,10 @@ def capture(camera=None,
 
         with contextlib.nested(
              _maintain_camera(panel, camera),
-             _applied_viewport_options(viewport_options, panel),
```

Comment:
Well I imagine someone might want to have a look at or apply a preset easily and would do so with `apply_options (**preset)` instead of triggering the individual commands.

If we'd separate out individual context managers again I'd still opt for having a `apply_options ()` available that can directly use results of `parse_view ()`.